pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -524,6 +533,38 @@ func createRedirectedTaskSpec(kubeclient kubernetes.Interface, ts *v1alpha1.Task
 	return ts, nil
 }
 
+// combineEnvVars takes a base slice of environment variables and a slice of environment
+// variables to overlay on top of the base slice, returning the result.
+func combineEnvVars(base []corev1.EnvVar, override []corev1.EnvVar) []corev1.EnvVar {
```

Comment:
looks like a great interface! i think this is a good candidate for moving into its own package with its own unit tests :stuck_out_tongue_winking_eye: :smile:

Reply:
Welp, I switched to a new function. Which I will move etc. =)